packages/core/src/universal/hooks/useSubscription/index.test.ts

```
@@ -0,0 +1,246 @@
+
+  onopen: (() => void) | null = null;
+  onmessage: ((event: { data: string }) => void) | null = null;
+  onerror: (() => void) | null = null;
+  onclose: (() => void) | null = null;
+  readyState = MockWebSocket.OPEN;
+  send = vi.fn();
+  close = vi.fn();
+
+  constructor(url: string, protocol: string) {
+    // Constructor implementation
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)